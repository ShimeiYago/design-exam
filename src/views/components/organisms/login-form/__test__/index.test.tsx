import { shallow, ShallowWrapper } from 'enzyme';
import { LoginForm, Props } from '..';

let wrapper: ShallowWrapper<Props>;

const props: Props = {
  onLogin: jest.fn(),
};

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<LoginForm {...props} />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
