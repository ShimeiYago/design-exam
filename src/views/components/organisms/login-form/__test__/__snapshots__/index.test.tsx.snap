// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<Fragment>
  <Spacing>
    <TextWrapper
      align="center"
      bold={true}
      size="xlarge"
    >
      ログイン
    </TextWrapper>
  </Spacing>
  <Spacing
    marginVertical="40px"
  >
    <Spacing
      marginBottom="24px"
    >
      <LabeledInputField
        label="メールアドレス"
        onChange={[Function]}
        value=""
      />
    </Spacing>
    <Spacing>
      <LabeledInputField
        label="パスワード"
        onChange={[Function]}
        value=""
      />
    </Spacing>
  </Spacing>
  <Spacing>
    <Button
      color="primary"
      fullWidth={true}
      onClick={[MockFunction]}
    >
      ログイン
    </Button>
  </Spacing>
</Fragment>
`;
